identity-apps-core/features/org.wso2.identity.apps.authentication.portal.server.feature/resources/web.xml.j2

```diff
@@ -366,8 +366,9 @@
         {"name": "org_name.do", "jsp": "/org_name.jsp", "url": "/org_name.do"},
         {"name": "org_discovery.do", "jsp": "/org_discovery.jsp", "url": "/org_discovery.do"},
         {"name": "select_org.do", "jsp": "/select_org.jsp", "url": "/select_org.do"},
-        {"name": "register.do", "jsp": "/self-registration.jsp", "url": "/register.do"},
-        {"name": "registration_error.do", "jsp": "/errors/registration_error.jsp", "url": "/registration_error.do"}
+        {"name": "register.do", "jsp": "/execution-flow.jsp", "url": "/register.do"},
```

Comment:
shall we remove the `.do` from the path

Reply:
Will send a separate PR as there are some other places we have to change. Eg:- Default value of the portal URL from branding preference